plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -8567,6 +8567,15 @@ public void testCreateTableWithDataLocationButObjectStoreDisabled()
                 "Data location can only be set when object store is enabled");
     }
 
+    @Test
+    @Override
+    public void testSetFieldMapKeyType()
+    {
+        // Iceberg doesn't support change a map 'key' column. Only map values can be changed.
+        assertThatThrownBy(super::testSetFieldMapKeyType)
+                .hasMessageContaining("Cannot alter map keys");
```

Comment:
optional 

```suggestion
.hasMessageContaining("Failed to set field type");
```

Reply:
It would be nice to verify if the message contains the reason. Changed to `Failed to set field type: Cannot alter map keys`.